src/Control/InitialisationMiddleware.php

```diff
@@ -48,6 +48,18 @@ class InitialisationMiddleware implements HTTPMiddleware
         'localhost',
     ];
 
+    /**
+     * Provide a value for the HTTP Strict Transport Security header.
+     * This header is only respected if you also redirect to SSL.
+     *
+     * Example configuration:
+     *
```

Comment:
Missing example?

Reply:
Derp, helps when you actually commit your local changes... fixed now.